### ISSUE TITLE
Comma after last argument in single line call creates fix failure

### ISSUE THREAD

**Issue by mabar** [squizlabs/PHP_CodeSniffer#3588]
**Describe the bug**
When comma is used after last argument in call, following sniffs report an issue and fail to fix it. It is caused by these sniffs colliding with each other.

**Code sample**
```php
call($arg,); // Reports Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma
call($arg, ); // Reports PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket
```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset>
  <rule ref="PSR2.Methods.FunctionCallSignature"/>
  <rule ref="Generic.Functions.FunctionCallArgumentSpacing"/>
</ruleset>
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcbf test.php ...`
3. See error message displayed
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
test.php                                              FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------
```

**Expected behavior**
Fixes the error without failure.

I believe it could be fixed in `Generic.Functions.FunctionCallArgumentSpacing` by allowing no space after comma, when comma is followed by right parenthesis. - `call($arg,)` should be okay.

The comma itself can be removed by `SlevomatCodingStandard.Functions.DisallowTrailingCommaInCall` sniff.

**Versions:**
 - OS: Windows 10 - WSL 2.0
 - PHP: 8.0.15
 - PHPCS: 3.6.2
 - Standard: custom

**Comment by jrfnl** [comment]
Duplicate of #3477 ?

**Comment by mabar** [comment]
Yes, it seems to be.
